common-rest/src/main/java/org/uniprot/api/common/repository/search/QueryResult.java

```diff
@@ -1,6 +1,7 @@
 package org.uniprot.api.common.repository.search;
```

Comment:
Should we rename the controller test class with prefix UnirefLight. e.g. UniRefLightSearchControllerIT and UniRefLightStreamControllerIT